django/db/backends/base/schema.py

```diff
@@ -1275,11 +1281,11 @@ def _delete_unique_sql(
             ) or
             (condition and not self.connection.features.supports_partial_indexes) or
             (include and not self.connection.features.supports_covering_indexes) or
-            (expressions and not self.connection.features.supports_expression_indexes)
-
+            (expressions and not self.connection.features.supports_expression_indexes) or
+            (db_tablespace and not self.connection.features.supports_tablespaces)
```

Comment:
I would revert this change and ignore the `db_tablespace` option when a database doesn't support it.

Reply:
I made `_get_index_tablespace_sql` return an empty string if the backend does not support custom tablespaces. Should make index parts using it behave the same as well.